pydy/codegen/tests/test_ode_function_generator.py

```diff
@@ -112,6 +114,25 @@ def test_set_linear_system_solver(self):
                                  linear_sys_solver=solver)
 
         assert g._solve_linear_system == solver
+    
+    def test_no_constants(self):
+        sys = models.multi_mass_spring_damper()
+        sym_rhs = sys.eom_method.rhs()
+        q = sys.coordinates
+        u = sys.speeds
+        p = [sm.Symbol('m0'), sm.Symbol('c0'), sm.Symbol('k0')]
+
+        #Equation sym_rhs with constants substituted with their values
+        sym_rhs2 = sm.MutableDenseMatrix([[sm.Function('v0')(sm.Symbol('t'))],
+        [sm.Mul(sm.Pow(1.0, sm.Integer(-1)),sm.Add(sm.Mul(sm.Integer(-1), 2.0, 
+        sm.Function('v0')(sm.Symbol('t'))), sm.Mul(sm.Integer(-1),
+        3.0, sm.Function('x0')(sm.Symbol('t')))))]])
+
+        rhs = generate_ode_function(sym_rhs, q, u, p)
+        rhs2 = generate_ode_function(sym_rhs2, q, u, p)
```

Comment:
The test should not have `p`. You should have to pass in `p` if there are no instances of those symbols in the rhs expressions.